### DISCUSSION TITLE
Uglify CSS

### DISCUSSION THREAD

**Discussion by IRediTOTO** [vercel/next.js#14423] [Help] [answered]
Hi, we all know about uglify js, but how about uglify css? :/ i tried to google for this but look like 404. Wont we have a better minify css if we can ugly it? 
Ex: `<h2 class=“a”>hello </h2>`
`a{font-size:14px}`
Just that super short... i think have a reason people dont do this

**Comment by Tylerian** [comment, marked as answer]
You must wait until #14197  gets approved and merged. It indirectly addresses this issue.

**Reply by IRediTOTO** [reply]
great, waiting for it. Thank you to made this.

**Reply by IRediTOTO** [reply]
Will your feature will apear in next version of Nextjs? Not hurry but very like this. Look so cool

**Reply by Tylerian** [reply]
I have no idea what their ETA is but I hope so :)

**Reply by IRediTOTO** [reply]
are still push this? Can I setup this manual by myself ?
